awx/main/management/commands/inventory_import.py

```diff
@@ -155,6 +155,8 @@ def command_to_json(self, cmd):
 
         proc = subprocess.Popen(cmd, stdout=subprocess.PIPE, stderr=subprocess.PIPE, env=env)
         stdout, stderr = proc.communicate()
```

Comment:
Earlier above in this file we have

```python
        env.pop('PYTHONPATH', None)  # default to none if no python_ver matches
        if os.path.isdir(os.path.join(venv_libdir, "python2.7")):
            env['PYTHONPATH'] = os.path.join(venv_libdir, "python2.7", "site-packages") + ":"
```

I fear this is going to have some majorly negative effects on the users who have a python3 venv installed, although I do not yet fully understand the mechanics of that.

It's something that I'm digging into a lot, because of the work on inventory plugins. We are going in the direction of allowing use of organization custom venv for inventory imports. That should open up some means of passing through the details. Until then, I figure this will be a major noted known issue that needs to be resolved after merge.

Reply:
Yep, this is a thing in my brain that needs solving, but I don't intend to solve it in this PR.

We'll definitely need a better plan for this when we support ansible venvs in py3 (which this PR doesn't totally do quite yet).